src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs

```diff
@@ -137,12 +139,25 @@ public string CreateAuthenticationRequestUrl()
         /// <returns>The uri to use for a redirect.</returns>
         public string CreateLogoutRequestUrl()
         {
+            EnsureTelemetryValues();
```

Comment:
You may want to consider adding telemetry on the cloned message. Otherwise turning it off, would result in the second message having the parameters.